plugin/src/main/java/org/owasp/benchmarkutils/score/BenchmarkScore.java

```diff
@@ -163,140 +118,47 @@ public class BenchmarkScore extends AbstractMojo {
     private static Map<String, CategoryResults> averageNonCommerciaToolResults = null;
     private static Map<String, CategoryResults> overallAveToolResults = null;
 
+    public static Configuration config;
+
     /**
      * Process the command line arguments that make any configuration changes.
      *
      * @param args - args passed to main().
-     * @return true if valid command line arguments provided. False otherwise.
      */
-    private static boolean processCommandLineArgs(String[] args) {
-
-        final String USAGE_MSG =
-                "Usage: -cf /PATH/TO/scoringconfigfile.yaml or -cr scoringconfigfile.yaml (where file is a resource)";
-
-        File yamlFile = null;
-        InputStream yamlFileStream = null;
-        if (args == null || args.length == 0) {
-            // No arguments is OK
-        } else if (args.length != 0 && args.length != 2) {
-            System.out.println(USAGE_MSG);
-        } else if (args.length == 2) {
+    private static void loadConfigFromCommandLineArguments(String[] args) {
+        if (args == null || args.length != 2) {
+            config = Configuration.fromDefaultConfig();
+        } else {
```

Comment:
Its not clear to me that this handles all the error cases properly. If args is != null and args.length is != 2 we should display the usage error. But it looks like in that case it just loads the default policy without showing an error.

Reply:
I'll add some test cases to make sure that there is an error message in the expected cases!

Reply:
So you've added some test cases/made changes to make sure there are proper error messages in all cases?

Reply:
I added the message, but did not write any test. Not sure, if this is an issue, currently time is a limiting factor to me 🙈

Reply:
@darkspirit510 - any chance you can write a few test cases for this before I merge?

Reply:
If you prefer, of course!

Reply:
please check the tests 😊